revocation/internal/crl/crl.go

```diff
@@ -18,35 +18,38 @@ package crl
 import (
 	"context"
 	"crypto/x509"
-	"crypto/x509/pkix"
 	"encoding/asn1"
 	"errors"
 	"fmt"
 	"math/big"
-	"slices"
 	"time"
 
 	"github.com/notaryproject/notation-core-go/revocation/crl"
+	"github.com/notaryproject/notation-core-go/revocation/internal/x509util"
 	"github.com/notaryproject/notation-core-go/revocation/result"
 	"golang.org/x/crypto/cryptobyte"
 )
 
+// RFC 5280, 5.3.1
+//
+//	CRLReason ::= ENUMERATED {
+//	  unspecified             (0),
+//	  keyCompromise           (1),
+//	  cACompromise            (2),
+//	  affiliationChanged      (3),
+//	  superseded              (4),
+//	  cessationOfOperation    (5),
+//	  certificateHold         (6),
+//	       -- value 7 is not used
+//	  removeFromCRL           (8),
+//	  privilegeWithdrawn      (9),
+//	  aACompromise           (10) }
 const (
-	// RFC 5280, 5.3.1
-	// CRLReason ::= ENUMERATED {
-	//   unspecified             (0),
-	//   keyCompromise           (1),
-	//   cACompromise            (2),
-	//   affiliationChanged      (3),
-	//   superseded              (4),
-	//   cessationOfOperation    (5),
-	//   certificateHold         (6),
-	//        -- value 7 is not used
-	//   removeFromCRL           (8),
-	//   privilegeWithdrawn      (9),
-	//   aACompromise           (10) }
+	// certificateHold
 	reasonCodeCertificateHold = 6
-	reasonCodeRemoveFromCRL   = 8
+
+	// removeFromCRL
+	reasonCodeRemoveFromCRL = 8
 )
```

Comment:
nit: you may put the comment at line 48 and 51 at the end of line 49 and 52.

Reply:
Updated.